semana-02/todo/add.py

```diff
@@ -0,0 +1,43 @@
+import json
+import uuid
+
+from file_manager import write_file, read_file
+
+def validate_task_struct(task: dict):
+    keys = task.keys()
+    required_keys = ["status", "desc"]
+    diff: set = required_keys - keys
+
+    if len(diff) > 0:
+        print(f"Error: {list(required_keys)} are required in data struct")
+        return True
+
+    if task["status"] not in ["in progress", "done", "not done"]:
+        print(f"Error: status must be 'in progress' or 'done'")
+        return True
+
+    if len(task["desc"]) == 0:
```

Comment:
Considerar utilizar task.get("desc") ao invés de task["desc"]. Neste caso, caso não exista a chave `desc` vai dar erro. Essa continua uma validação relevante, portanto você poderia fazer algo como:
```suggestion
    if task.get("desc") is None or len(task.get("desc")) == 0:
```